Source/WebCore/rendering/RenderHighlight.cpp

```diff
@@ -57,35 +57,37 @@ RenderRangeIterator::RenderRangeIterator(RenderObject* start)
 
 RenderObject* RenderRangeIterator::current() const
 {
-    return m_current;
+    return m_current.get();
 }
 
 RenderObject* RenderRangeIterator::next()
 {
-    RenderObject* currentSpan = m_spannerStack.isEmpty() ? nullptr : m_spannerStack.last()->spanner();
-    m_current = m_current->nextInPreOrder(currentSpan);
+    CheckedPtr currentSpan = m_spannerStack.isEmpty() ? nullptr : m_spannerStack.last()->spanner();
+    CheckedPtr checkedSpan = currentSpan.get();
```

Comment:
We don't normally use `checked` or `protected` in variable names, and we try to keep the names short.